awx/main/management/commands/run_rsyslog_configurer.py

```diff
@@ -26,6 +27,7 @@ def handle(self, *arg, **options):
                 conn.listen("rsyslog_configurer")
                 # reconfigure rsyslog on start up
                 reconfigure_rsyslog()
+                connection.close()
                 for e in conn.events(yield_timeouts=True):
                     if e is not None:
                         logger.info("Change in logging settings found. Restarting rsyslogd")
```

Comment:
I still think it would be good to close again after `TaskWorker.run_callable(body)`